### PR TITLE
Tanka

### PR DESCRIPTION
**Pull request by rolandknight**
tanka kubernetes installer (similar to helm)

### CODE REVIEW

File: tanka.hcl [hunk 1]
```diff
@@ -0,0 +1,31 @@
+description = "Flexible, reusable and concise configuration for Kubernetes."
+binaries = ["tk"]
+test = "tk --help"
+
+linux {
```
Comment (alecthomas) [inline tanka.hcl:5]: FYI if you have two platform blocks that are identical, as these two are, you can just pull them out to the top level.

**Comment by alecthomas** [comment]
Can you please clean up your fork so that commits don't keep accumulating?

**Comment by alecthomas** [comment]
Thanks!

**Comment by rolandknight** [comment]
Done. Created another clean pull request that simplifies tanka and adds
jsonnet-bundler (it's a tool that tanka uses BTW)

**Pull request merged**